test/e2e/provisioning/all_e2e_test.go

```diff
@@ -744,6 +746,42 @@ func TestHetznerProvisioningE2E(t *testing.T) {
 	runScenarios(t, selector, params, HZManifest, fmt.Sprintf("hz-%s", *testRunIdentifier))
 }
 
+// TestVultrBareMetalProvisioning - a test suite that exercises Vultr provider with a bare metal instance
+// by requesting nodes with different combination of container runtime type, container runtime version and the OS flavour.
+func TestVultrBareMetalProvisioningE2E(t *testing.T) {
```

Comment:
Those tests need to exist in prow as well: 
https://github.com/kubermatic/machine-controller/tree/master/.prow

However, for the time being, we don't have the infrastructural requirements to run the e2e tests for Vultr.